snippets/dart_snippets/lib/closed_channel.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:breez_sdk/breez_sdk.dart';
+import 'package:breez_sdk/bridge_generated.dart';
+
+Future<PrepareSweepResponse> prepareSweep({required int feeRate}) async {
+  // ANCHOR: prepare-sweep
+  final satPerVbyte = feeRate;
+  PrepareSweepRequest req = PrepareSweepRequest(
+    toAddress: "bc1..",
+    satPerVbyte: satPerVbyte,
+  );
+  final resp = await BreezSDK().prepareSweep(req: req);
+  // ANCHOR_END: prepare-sweep
+  return resp;
+}
+
+Future<SweepResponse> sweep(
+    {required int feeRate, required String toAddress}) async {
+  // ANCHOR: sweep
+  final satPerVbyte = feeRate;
+  SweepRequest req = SweepRequest(
```

Comment:
Requires reformatting, don't commit these changes using GitHub UI.

```suggestion
Future<SweepResponse> sweep(
    {required int satPerVbyte, required String toAddress,}) async {
  // ANCHOR: sweep
  SweepRequest req = SweepRequest(
```

Reply:
Done